liquibase-standard/src/main/java/liquibase/database/jvm/JdbcConnection.java

```diff
@@ -609,7 +580,11 @@ public boolean supportsBatchUpdates() throws DatabaseException {
         }
     }
 
-    private static class PatternPair {
+    /**
+     *  @deprecated this class has been moved to {@link ConnectionPatterns}, use that one instead.
+     */
+    @Deprecated
+    public static class PatternPair {
```

Comment:
Can this class be deleted? If it was private before, it will not be a breaking change.

Reply:
Oh! I missed that it was private. I'll remove it. Thanks!